stormbench/pom.xml

```diff
@@ -34,5 +34,46 @@
         </property>
       </activation>
     </profile>
+
```

Comment:
thanks @manuzhang  In the storm/streaming pom file, there is a dependency(org.apache.kafka) which need to be specified the scala version. So I set the scala version for the storm.